<launch>
	<!-- Episodic Memory -->
    <arg name="neo4j_path" default="$(env NEO4J_HOME)" />

	<node name="episodic_memory_node" pkg="episodic_memory" type="episodic_memory_node.py" respawn="false" output="screen">
        <param name="config_file" value="$(find episodic_memory)/config/em_conf.yaml" />
        <param name="neo4j_memory" command="$(arg neo4j_path)/bin/neo4j start"/>
	</node>
</launch>
